releases/release-1.30/release-notes/maps/pr-118389-map.yaml

```
@@ -0,0 +1,3 @@
+pr: 118389
+releasenote:
+  text: ""
```

How about we include the release notes for these PRs ourselves?
Something like -
```suggestion
  text: "Scheduler can now automatically reschedule unschedulable Pods in response to the changes in node annotations"
```

---

So this one looks like it has the `release-notes-none` label: https://github.com/kubernetes/kubernetes/pull/118389 I don't know if we want to add notes if the reviewer explicitly opted out, wdyt?

---

After looking through previous release notes, it seems common for NONE to be used, so I think in this case NONE is fine. But it is worth exploring in the future for those without the tag, an option to create one as @fykaa suggested.